core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -553,10 +553,10 @@ object ConsumerGroupCommand extends Logging {
         }.toMap
       } else if (opts.options.has(opts.resetByDurationOpt)) {
         val duration = opts.options.valueOf(opts.resetByDurationOpt)
-        val durationParsed = DatatypeFactory.newInstance().newDuration(duration)
-        val now = new Date()
-        durationParsed.negate().addTo(now)
-        val timestamp = now.getTime
+        val durationParsed = Duration.parse(duration)
+        val now = Instant.now()
+        durationParsed.negated().addTo(now)
+        val timestamp = now.toEpochMilli
```

Comment:
Is this code covered by any tests? Duration is immutable, so I don't think this code would work.

What do you think about:

```suggestion
        val timestamp = now.minus(durationParsed).toEpochMilli()
```

(also applies to the same change below)

Reply:
> Is this code covered by any tests?

Yes. Jenkins failed (I did not run the tests locally before I pushed...) -- Same issue in `StreamsResetter`.